### DISCUSSION TITLE
When I set `routerMode: 'history'`, how do I access the subpages directly

### DISCUSSION THREAD

**Discussion by rakuyoMo** [docsifyjs/docsify#1858] [Q&A] [answered]
I'm using **Docsify + VPS** to build a document system on my company intranet. One of the pages needs to be set up for public access, and I'm going to use nginx for that, so I can't carry `#` on the url, I have to use `history` for `routerMode`.

But when I configured it, I found that I can't access this page directly (type the path of the subpage directly in the browser, for example `https://www.some-doc.com/privacy`), it prompts **404**, page not found.

Is there any way to solve this problem? How do I access a page directly in history mode?

**Comment by trusktr** [comment, marked as answer]
Hello, it isn't related to Docsify, but with any static site that does not use hash routing, you'll have to configure your server to send the same HTML (as for the root `/` path) for any path. Your server currently responds with a 404 instead of sending back the index.html.

Here's where to start on that with Nginx:

https://www.google.com/search?q=nginx+redirect+all+paths+to+index.html

**Reply by rakuyoMo** [reply]
Thank you very much, my problem is solved
